File: apps/frontend-v2/fix_footer.py
import os

filepath = r'c:\Users\User\Documents\Sync Pcloud\Professionnel\Dev\sosplanete-v2\apps\frontend-v2\src\app\dashboard\organization\page.tsx'

with open(filepath, 'r', encoding='utf-8') as f:
    content = f.read()

# Condense footer widths
content = content.replace('w-14 text-right', 'w-12 text-right')
content = content.replace('w-16 text-right', 'w-14 text-right')
content = content.replace('gap-5 shrink-0', 'gap-3 shrink-0')

with open(filepath, 'w', encoding='utf-8', newline='\n') as f:
    f.write(content)

File: apps/frontend-v2/fix_footer.js
const fs = require('fs');
const path = 'c:\\Users\\User\\Documents\\Sync Pcloud\\Professionnel\\Dev\\sosplanete-v2\\apps\\frontend-v2\\src\\app\\dashboard\\organization\\page.tsx';
let content = fs.readFileSync(path, 'utf8');

content = content.replace(/w-14 text-right/g, 'w-12 text-right');
content = content.replace(/w-16 text-right/g, 'w-14 text-right');
content = content.replace(/gap-5 shrink-0/g, 'gap-3 shrink-0');

fs.writeFileSync(path, content, 'utf8');
console.log('Done');
